intranet/static/js/morningbus.js

```diff
@@ -0,0 +1,114 @@
+
+$(function() {
```

Comment:
Could we eliminate redundancies between this file and `intranet/static/js/bus.js`?